### ISSUE TITLE
JSON::ParseError is raised when Nexmo REST service is not available

### ISSUE THREAD

**Issue by nevalla** [Vonage/vonage-ruby-sdk#4]
When the Nexmo rest service is not available, JSON::ParseError is raised in [Client.send_message(data)](https://github.com/timcraft/nexmo/blob/master/lib/nexmo.rb#L23) method

``` ruby
JSON::ParserError (756: unexpected token at '<html>
<head>
<meta http-equiv="Content-Type" content="text/html;charset=ISO-8859-1"/>
<title>Error 500 Server Error</title>
</head>
<body>
<h2>HTTP ERROR: 500</h2>
<p>Problem accessing /sms/json. Reason:
<pre>    Server Error</pre></p>
<hr /><i><small>Powered by Jetty://</small></i>




















</body>
</html>
')
```

**Comment by timcraft** [comment]
Good catch. Definitely needs some kind of fix.

**Comment by timcraft** [comment]
Hopefully addressed with 7b9c8108424547e891a8adbdc936d77d8273de2e, version 0.3.0 will return a failure object for unexpected HTTP responses and the underlying response object is accessible.
